packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx

```diff
@@ -30,7 +30,7 @@ const EAHomeCommunityPosts = ({classes}:{classes: ClassesType}) => {
   return (
     <AnalyticsContext pageSectionContext="communityPosts">
       <SingleColumnSection>
-        <SectionTitle title="Community posts"></SectionTitle>
+        <SectionTitle title="Posts about the community"></SectionTitle>
```

Comment:
Hmm, I actually think I prefer the original wording, this feels a bit verbose

Reply:
yeah I agree it's pretty verbose :/ the reason I wanna change it is so many people in user interviews have no idea what "Community posts" mean and how they're different from the other posts (think they think all posts are community posts cause they're written by the community?). This aimed to clarify that. Another option I had was "Posts tagged community", which could be more clear if the concept of "tagging" feels clear to people